gtk4/src/auto/actionable.rs

```diff
@@ -118,6 +116,33 @@ impl<O: IsA<Actionable>> ActionableExt for O {
             )
         }
     }
+
+    fn connect_property_action_target_notify<F: Fn(&Self) + 'static>(
+        &self,
+        f: F,
+    ) -> SignalHandlerId {
+        unsafe extern "C" fn notify_action_target_trampoline<P, F: Fn(&P) + 'static>(
+            this: *mut ffi::GtkActionable,
+            _param_spec: glib::ffi::gpointer,
```

Comment:
Why ignore the pspec?

Reply:
ah, somehow gir does that all the time for any pspec used in a trampoline

Reply:
That seems like a bug, can you create an issue at least?

Reply:
Sure, https://github.com/gtk-rs/gir/issues/1042